apps/meteor/tests/end-to-end/api/livechat/19-business-hours.ts

```diff
@@ -321,6 +321,21 @@ describe('LIVECHAT - business hours', function () {
 			expect(latestAgent.openBusinessHours).to.be.an('array').of.length(0);
 			expect(latestAgent.statusLivechat).to.be.equal(ILivechatAgentStatus.NOT_AVAILABLE);
 		});
+
+		it('should create a custom business hour which is closed by default, but a bot agent shouldnt be affected', async () => {
+			const bot = await createUser({ roles: ['bot', 'livechat-agent'] });
+			const creds = await login(bot.username, password);
+			await makeAgentAvailable(creds);
+
+			const { department } = await createDepartmentWithAnOnlineAgent({ user: bot, credentials: creds });
+
+			await createCustomBusinessHour([department._id], false);
+
+			const latestAgent: ILivechatAgent = await getMe(creds);
+			expect(latestAgent).to.be.an('object');
+			expect(latestAgent.openBusinessHours).to.be.an('array').of.length(0);
+			expect(latestAgent.statusLivechat).to.be.equal(ILivechatAgentStatus.AVAILABLE);
+		});
```

Comment:
Shouldn't we create a test case with the BH feature off, and then turn it on and check if it's behaving properly? According to what we have in the changeset, it looks like this only happens when the feature is turned on. Or is this already covered with the above test case?